src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
+    private final int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball{"
+                + "color='" + color + '\''
+                + "number=" + number
+                + "}";
+    }
+}
+

```

Comment:
https://github.com/anna-halas/jv-lottery/blob/lottery/checklist.md#please-dont-add-redundant-empty-lines-to-your-code
```suggestion

```